docs/content/guides/dagster/code-references.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Linking to assets' code with code references | Dagster Docs
```

Comment:
The possessive is a weird stylization in a title. I wonder if we can circumvent it altogether. We already use "asset definition" below anyways.

```suggestion
title: Linking to asset definition code with code references | Dagster Docs
```